projects/canopy/src/lib/card/card.component.ts

```diff
@@ -25,11 +25,8 @@ export class LgCardComponent implements AfterContentInit {
   @ContentChild(LgCardContentComponent)
   lgCardContentComponent: LgCardContentComponent;
 
-  cardContent: QueryList<LgCardContentComponent>;
+  @ContentChild(LgCardFooterComponent)
+  lgCardFooterComponent: LgCardFooterComponent;
 
-  ngAfterContentInit() {
-    if (this.lgCardHeaderComponent && this.lgCardContentComponent) {
-      this.lgCardHeaderComponent.hasContent = true;
-    }
-  }
+  cardContent: QueryList<LgCardContentComponent>;
```

Comment:
I might be wrong but I suspect none of these content children are used anymore and could all go away.

Reply:
Yep, that is correct. 
Done.